test_add_remove_ldap_role_permission.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+from cassandra import Unauthorized
+
+from longevity_test import LongevityTest
+from sdcm.cluster import BaseNode
+from sdcm.sct_events.system import InfoEvent
+from sdcm.utils.ldap import LdapServerType, LDAP_USERS, LDAP_PASSWORD, LdapUtilsMixin
+
+
+class AddRemoveLdapRolePermissionTest(LongevityTest, LdapUtilsMixin):
+
+    def test_add_remove_ldap_role_permission(self):  # pylint: disable=too-many-statements
+        """
+        Scenario:
+        1. Create new user in Scylla
+        2. Create new role in Ldap containing the new user as a member.
+        3. Verify this new user is unauthorised on resources (create a keyspace).
+        4. Create a new super-user role in Scylla.
+        5. Create a new super-user role in Ldap, with the new user as a member.
+        6. Verify this new user is authorised on resources.
+        7. Modify Ldap role to remove membership of new user.
+        8. Verify the new user is unauthorised on resources (create a table).
+        9. Do clean-up: delete Ldap Role, Scylla roles and keyspace/tables.
+
+        """
+        node: BaseNode = self.db_cluster.nodes[0]
+        stress_queue = []
+        background_write_cmd = "cassandra-stress write no-warmup cl=QUORUM duration=10m -schema 'replication(factor=3)' " \
+                               "-mode cql3 native -rate threads=2 -pop seq=1..1002003 -log interval=5"
```

Comment:
i would prefer to have this background c-s command in the test `.yaml` configuration file

Reply:
fixed.